src/component/registry/css-registry.ts

```diff
@@ -32,30 +32,64 @@ export class CssRegistry {
   /**
    * Get the CSS class names for a specific HTML element
    *
-   * @param bpmnElementId the BPMN id of the HTML element from the DOM
+   * @param bpmnElementId the BPMN ID of the HTML element from the DOM
    * @return the registered CSS class names
    */
   getClassNames(bpmnElementId: string): string[] {
     return Array.from(this.classNamesByBpmnId.get(bpmnElementId) ?? []);
   }
 
+  /**
+   * Get all BPMN element IDs with at least one CSS class on its HTML element.
```

Comment:
**suggestion:** this is not always true. If I remove all CSS classes, the id is still available and returns an empty array.
As this is an internal API, we may not put too much effort on this. I prefer no documentation instead of a partially true documentation.